src/environmentbase/template.py

```diff
@@ -537,6 +506,76 @@ def add_cloudwatch_alarm(self,
 
         return self.add_resource(alarm)
 
+    @staticmethod
+    def image_map_name(image_label):
+        """
+        Each image map is a concatenation of a custom label and a prefix. This function
+        provides a consistant way to get the correct image map name from the custom label.
+        @returns string
+        """
+        return Template.IMAGE_MAP_PREFIX + image_label.title()
```

Comment:
I think it would be better to leave `.title()` off, cause it's hard to reverse engineer the image_label after it has been titled.